README.md

```diff
@@ -289,7 +289,7 @@ yo chisel:page "Home" "About Us" "Contact Us" "News"
 When you have the basic setup done, you can start development. To re-compile Twig, SCSS and JavaScript files in real time you can use default task. Type
 
 ```bash
-gulp
+npm run dev
```

Comment:
I used `npm run dev` to follow the Chisel blog post, I'll update PR in the morning.